src/convertToObject.js

```diff
@@ -6,7 +6,48 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrSoursString = sourceString;
+  const result = {};
```

Comment:
The variable name `result` is too generic. Consider using a more descriptive name like `styleObject` to improve code readability.